application/org.openjdk.jmc.ui/src/main/java/org/openjdk/jmc/ui/misc/QuantityKindProposal.java

```diff
@@ -57,7 +57,13 @@
 import org.openjdk.jmc.common.unit.QuantityConversionException;
 
 public class QuantityKindProposal {
-	private static final IContentProposalProvider EMPTY_PROPOSAL = new SimpleContentProposalProvider(new String[0]);
+	private static final int POPUP_ROW_HEIGHT = 15;
```

Comment:
How can you know this would be the size? This is font-related, no?

Reply:
it's used for calculating a popup width and height : adapter.setPopupSize(new Point(POPUP_MAX_WIDTH, popupHeight)); I've not find a way how to get a font size. So I've estimated nice fit for the current setup, as previously was hardcoded everything. Do you think the name is not so good ?

Reply:
I think this is a pseudo-implementation that may just be confusing. Personally I'd prefer the previous fixed size solution - that one will be understandable to all users, no matter the font settings. This one runs the risk of feeling slightly off / buggy to users depending on the font settings.

Reply:
I think it's quite reasonable, as the benefit of having it kind of "dynamic" is minimal. I'd personally also prefer the fixed size. I'll change it.

Reply:
corrected!